src/components/Navbar/index.js

```diff
@@ -23,7 +23,7 @@ const COLOR_THEME_SECONDARY = {
 
 export default class Navbar extends Component {
   static propTypes = {
-    currentPage: PropTypes.oneOf(["home", "company", "blog"]),
+    currentPage: PropTypes.oneOf(["home", "company", "blog", "careers"]),
```

Comment:
Why do you need this change?

Reply:
A console error appear connect with the miss of careers word on array.
`Warning: Failed prop type: Invalid prop `currentPage` of value `careers` supplied to `Navbar`, expected one of ["home","company","blog"].`

Reply:
Ok 👍